lib/Dialect/ESI/runtime/CMakeLists.txt

```diff
@@ -175,6 +163,14 @@ if (XRT_PATH)
     ${ESIRuntimeLibDirs}
     ${XRT_PATH}/lib
   )
+  get_filename_component(XRT_SO ${XRT_PATH}/lib/libxrt_coreutil.so REALPATH)
+  install(FILES
+      ${XRT_SO}
+      ${XRT_PATH}/lib/libxrt_coreutil.so.2
+      ${XRT_PATH}/lib/libxrt_coreutil.so
+    DESTINATION lib
+    COMPONENT ESIRuntime
+  )
```

Comment:
Yeah, I'm choosing to package the shared object rather than rely on a locally installed one so if someone ends up with a binary runtime distribution, they won't run into dynamic linking issues if they don't install the xrt runtime. Ideally, we'd have a plugin scheme instead. But you're right, it shouldn't be in this PR. I'll remove it.